capitalize.rb

```diff
@@ -0,0 +1,7 @@
+require './base_decorator'
+
+class Capitalize < Basedecorator
+  def correct_name
+    super.capitalize
+  end
```

Comment:
- [ ] Kindly make sure you update this method in the way it Capitalizes the first letter of the given name for the project requirements.